src/Service/BasicService.php

```diff
@@ -189,6 +189,7 @@ public function run($HAL = false)
         $object = ($this->requestObject instanceof BasicRequest) ? $this->requestObject->getObject() : [];
         $body = [];
         $option = [];
+        $option['headers']['Authorization'] = 'Bearer ' . $GLOBALS['id_token'];
```

Comment:
@pooja-chawla Ideally this should be conditionally handled only if the id_token is set global.